Code/Framework/AzFramework/Platform/Windows/AzFramework/Components/NativeUISystemComponent_Windows.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/Application/Application.h>
+#include <AzFramework/Components/NativeUISystemComponent.h>
+
+#include <AzFramework/Application/Application_Windows.h>
+#include <AzFramework/Input/Devices/Mouse/InputDeviceMouse_Windows.h>
+#include <AzFramework/Input/Devices/Keyboard/InputDeviceKeyboard_Windows.h>
+#include <AzFramework/Input/Devices/Gamepad/InputDeviceGamepad_Windows.h>
+#include <AzFramework/Windowing/NativeWindow_Windows.h>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+namespace AzFramework
+{
+    class WindowsApplicationImplFactory 
+        : public Application::ImplementationFactory
+    {
+    public:
+        Application::Implementation* Create() override
+        {
+            return aznew ApplicationWindows();
+        }
```

Comment:
This function should return a unique_ptr.
```suggestion
        AZStd::unique_ptr<Application::Implementation> Create() override
        {
            return aznew ApplicationWindows();
        }
```